Jenkinsfile

```diff
@@ -45,7 +45,9 @@ def sanitized_JOB_NAME = JOB_NAME.toLowerCase().replaceAll('/', '-').replaceAll(
 
 def rpm_test_pre = '''export PDSH_SSH_ARGS_APPEND="-i ci_key"
                       nodelist=(${NODELIST//,/ })
-                      scp -i ci_key src/tests/ftest/data/daos_server_baseline.yaml jenkins@${nodelist[0]}:/tmp
+                      scp -i ci_key src/tests/ftest/data/daos_server_baseline.yaml \
+				    utils/config/daos_agent.yml \
```

Comment:
This shouldn't come from the source pool but rather wherever it's installed in a production (i.e. installed from RPM) environment -- `/usr/etc/` (should be `/etc/` eventually).  I know that the `daos_server_baseline.yaml` is coming from the source pool but that's due to an open ticket to get all of the Functional testing into an RPM which is still yet to be worked.

Reply:
So the scons file places it in install/etc when you build. so I'm assuming it will be under /etc

Reply:
It will be in `/usr/etc` for the moment.  It is still to be worked out having `$PREFIX/etc` placed in `/etc` instead of `/usr/etc`.